videoframes.py

```diff
@@ -206,6 +221,22 @@ def change_stream_quality(self, quality):
         self.stream.change_stream_quality(quality)
         self.player.play()
 
+    def open_stream_in_browser(self, event):
+        url = self.stream.url.lower()
```

Comment:
No need to cast to lower anymore. All urls are now stored as lower-case when the user enters it.